### DISCUSSION TITLE
Nombre de Dominio

### DISCUSSION THREAD

**Discussion by ZafkielEdt** [FrancoGL/ISPC-FullStack#57] [Polls]
Aquí iremos publicando las ideas de nombres de dominio:

**Comment by veroivk** [comment]
Antes de seguir, quisiera que a todos nos quede claro este tema. Para el módulo fullstack no es necesario tener un dominio, sino linkear nuestra empresa ficticia donde sí estará alojado WordPress (ver trabajo grupal de ese módulo) o en Github (Settings u Options). Entendí bien?

**Reply by ZafkielEdt** [reply]
Para la parte de frontend no, simplemente se puede levantar con GitHub pages,  Vercel o neflify.

**Reply by federico42o** [reply]
si etendiste bien, es asi
